bundles/org.eclipse.e4.emf.xpath/src/org/eclipse/e4/emf/xpath/XPathContext.java

```diff
@@ -42,19 +46,35 @@ public interface XPathContext {
 	 *            required type
 	 * @return Object found
 	 */
-	Object getValue(String xpath, Class<?> requiredType);
+	<T> T getValue(String xpath, Class<T> requiredType);
```

Comment:
Since the method signature is the same after generic erasure this is a binary and source compatible change.
But as far as I can tell, this method and its overload is not used within the SDK at all, besides tests.